web/src/app/shared/models/user.model.ts

```diff
@@ -7,6 +7,7 @@ export class UserModel {
   username: string;
   avatarUrl: string = '';
   url: string = '';
+  following?: string[] = [];
```

Comment:
I don't think this is needed here? We use `ProfileModel` in our application, this is just for the authentication with GitHub OAuth - if I remember correctly

Reply:
ok got it